### ISSUE TITLE
Files are not updated after opening FilePicker for the first time

### ISSUE THREAD

**Issue by amagh** [DroidNinja/Android-FilePicker#82]
The FilePicker isn't updating the list of files it shows after opening the FilePicker for the first time. If I open the FilePicker and there are no valid files, they still wont show even after adding them.

For example: I have no *.mp3 files. Then I open the FilePicker to try to select an *.mp3 file, but find out that I forgot to transfer it to the external storage first. I go on my computer and transfer the file and again open FilePicker. It still says there are no *.mp3 files. I have to reboot the device before the *.mp3 file I transferred shows.

This also occurs if I rename a file. If I have a music.mp3 on my external, and then I open FilePicker, it shows. Then I later rename the file to song.mp3, FilePicker still shows the filename as music.mp3 and returns the incorrect file location because music.mp3 no longer exists.

I can see the changes and new files in a third-party file browser app like SolidExplorer no problem.

Thanks!

**Comment by wenwenwen888** [comment]
I have the same problem

**Comment by wenwenwen888** [comment]
@amagh hey man , I have just solved the problem .
when you add or modify a file . Just execute this line of code
```
public static void sendBroadcastToMedia(Context context, String filePath) {
        Intent intent = new Intent(Intent.ACTION_MEDIA_SCANNER_SCAN_FILE);
        Uri uri = Uri.fromFile(new File(filePath));
        intent.setData(uri);
        context.sendBroadcast(intent);
    }
```
Notice the media to update the database .

**Comment by DroidNinja** [comment]
I think this is not filepicker fault. Filepicker uses MediaStore database for fetching list of files. If device has not updated the default Mediastore database, then, it will not show in the picker.

**Comment by DroidNinja** [comment]
There was issue for audio type. Fixed in 2.0.3
